library/core/src/intrinsics.rs

```diff
@@ -1803,6 +1803,29 @@ extern "rust-intrinsic" {
     #[rustc_nounwind]
     pub fn fmaf128(a: f128, b: f128, c: f128) -> f128;
 
+    /// Returns `a * b + c` for `f32` values.
```

Comment:
```suggestion
    /// Returns `a * b + c` for `f32` values, non-deterministically executing either a
    /// fused multiply-add or two operations with rounding of the intermediate result.
```
Same for the other one.

Reply:
Done.